starter-code/pseudocode.txt

```diff
@@ -73,5 +73,5 @@ func bonusFunction(var n as int)
     return total
 end func
 
-Running time: 
-Explanation: 
+Running time: Big O(n + n^2) = Big O(n^2)
+Explanation: The func has multiple nested loops and is dependent on how large or small of a number the input is for the last while loop
```

Comment:
There's 3 nested loops here - the while loop inside the for-k look, inside the for-i loop. Three nested loops leads to a O(n^3) running time